components/NodeBox/NodeBox.tsx

```diff
@@ -21,14 +22,14 @@ function NodeBox({
   pointed,
   color,
   name,
-  arrow = false,
-  label = ''
+  arrow,
+  label = '',
+  arrowOrientation,
 }: Props) {
   return (
     <div>
       <ValueBox color={color} name={name} label={label} />
-      {pointer && <PointerBox color={color} pointed={true} arrow={arrow} />}
```

Comment:
naming of this props is not precise, what does arrow mean? is it boolean?
why you removed arrow = false?

Reply:
Arrow means arrow and yes, this is boolean. it simply indicates the presence of an arrow in a specific component